package.json

```diff
@@ -16,37 +16,47 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "antd": "^4.8.0",
+    "@babel/plugin-transform-regenerator": "^7.12.1",
+    "babel-plugin-import": "^1.13.1",
     "eslint-config-airbnb": "^18.2.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.4.0",
     "eslint-plugin-promise": "^4.2.1",
     "express": "^4.16.4",
+    "jest": "^26.6.3",
+    "node-fetch": "^2.6.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "react-test-renderer": "^17.0.1"
+    "react-redux": "^7.2.2",
+    "react-test-renderer": "^17.0.1",
+    "redux": "^4.0.5",
+    "redux-devtools-extension": "^2.13.8",
+    "redux-mock-store": "^1.5.4",
+    "redux-thunk": "^2.3.0",
+    "reselect": "^4.0.0"
   },
   "engines": {
     "node": ">=4.3.2"
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.12.1",
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
-    "babel-jest": "^24.1.0",
+    "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.0",
     "css-loader": "^3.2.0",
-    "enzyme": "^3.8.0",
-    "enzyme-adapter-react-16": "^1.7.1",
     "eslint": "^5.14.1",
     "eslint-loader": "^2.1.1",
     "eslint-plugin-react": "^7.12.1",
+    "fetch-mock": "^9.10.7",
     "file-loader": "^1.1.11",
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.2.0",
-    "jest": "^24.1.0",
+    "less": "^2.3.1",
+    "less-loader": "^4.0.6",
```

Comment:
is it needed since we are using sass-loader?

Reply:
removed less-loader